test/dlc_tests/conftest.py

```diff
@@ -1002,47 +993,23 @@ def skip_p5_tests(request, ec2_instance_type):
 
 
 @pytest.fixture(autouse=True)
-def skip_pt20_cuda121_tests(request):
+def skip_serialized_release_pt_test(request):
     if "training" in request.fixturenames:
         image_uri = request.getfixturevalue("training")
     elif "pytorch_training" in request.fixturenames:
         image_uri = request.getfixturevalue("pytorch_training")
     else:
         return
 
-    skip_dict = {"==2.0.*": ["cu121"]}
+    skip_dict = {
+        ">=2.1,<2.4": ["cpu", "cu121"],
+    }
     if _validate_pytorch_framework_version(
-        request, image_uri, "skip_pt20_cuda121_tests", skip_dict
+        request, image_uri, "skip_serialized_release_pt_test", skip_dict
     ):
-        pytest.skip("PyTorch 2.0 + CUDA12.1 image doesn't support current test")
-
-
-@pytest.fixture(autouse=True)
-def skip_pt21_test(request):
```

Comment:
assuming you searched codebase for removed fixtures before removing? if so feel free to resolve this

Reply:
yep